### PR TITLE
Feature/contat us view controller

### PR DESCRIPTION
**Pull request by Slavitor**
Contact us View Controller

### CODE REVIEW

File: Showcase-iOS/Contact Us/View/ContactUsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        return OfficeViewModel(with: Office(with: mockJhbOffice))
+    }
+}
+
+//MARK: CollectionView extension
+
+extension ContactUsViewController: UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout{
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        return CGSize(width: collectionView.frame.width, height: 236)
```
Comment (pjwelcome) [inline Showcase-iOS/Contact Us/View/ContactUsViewController.swift:59]: magic number? where did you get this number

**Review by pjwelcome — CHANGES_REQUESTED** [review]
Merge conflicts, Please resolve this

File: Showcase-iOS/Contact Us/View/ContactUsViewController.swift (hunk 1 above)
Comment (Slavitor) [reply Showcase-iOS/Contact Us/View/ContactUsViewController.swift:59]: That's the height of the nib.

Comment (pjwelcome) [reply Showcase-iOS/Contact Us/View/ContactUsViewController.swift:59]: should the height not be determined by the collection view cell? and should be dynamic. If you happy to refactor later. Please make a ticket. But also need to resolve merge conflicts

Comment (Slavitor) [reply Showcase-iOS/Contact Us/View/ContactUsViewController.swift:59]: I will refactor later. I'm fixing the conflicts now. Will add a ticket also.

**Review by pjwelcome — APPROVED** [review]

**Pull request merged**